services_app/src/main/java/org/opendatakit/services/sync/actions/fragments/LoginFragment.java

```diff
@@ -166,6 +167,7 @@ private void setNewCredentials() {
       Map<String, String> properties = new HashMap<String, String>();
       properties.put(CommonToolProperties.KEY_AUTHENTICATION_TYPE,
           getString(R.string.credential_type_username_password));
+      properties.put(CommonToolProperties.KEY_CURRENT_USER_STATE, UserState.AUTHENTICATED_USER.toString());
```

Comment:
Check what `UserState.AUTHENTICATED_USER.toString()` prints you.
https://stackoverflow.com/a/13291109 refer to this one for better understanding.